lib/checks/keyboard/focusable-disabled-evaluate.js

```diff
@@ -15,21 +15,28 @@ function focusableDisabledEvaluate(node, options, virtualNode) {
     return true;
   }
 
-  const relatedNodes = tabbableElements.reduce((out, { actualNode: el }) => {
-    const nodeName = el.nodeName.toUpperCase();
-    // populate nodes that can be disabled
-    if (elementsThatCanBeDisabled.includes(nodeName)) {
-      out.push(el);
-    }
-    return out;
-  }, []);
+  const relatedNodes = tabbableElements.filter(vNode => {
+    return elementsThatCanBeDisabled.includes(vNode.props.nodeName);
+  });
 
-  this.relatedNodes(relatedNodes);
+  this.relatedNodes(relatedNodes.map(vNode => vNode.actualNode));
```

Comment:
One of these days I'm going to create a PR to just allow virtual nodes here... Not today though.